support-frontend/assets/pages/paper-subscription-checkout/helpers/options.ts

```diff
@@ -14,24 +14,43 @@ import {
 	ActivePaperProductTypes,
 	Everyday,
 } from 'helpers/productPrice/productOptions';
+import * as storage from 'helpers/storage/storage';
 import type { DateYMDString } from 'helpers/types/DateString';
 import { getQueryParameter } from 'helpers/urls/url';
 import { formatMachineDate } from 'helpers/utilities/dateConversions';
 import { getHomeDeliveryDays } from 'pages/paper-subscription-checkout/helpers/homeDeliveryDays';
 import { getVoucherDays } from 'pages/paper-subscription-checkout/helpers/voucherDeliveryDays';
 
 function getProductOption(): PaperProductOptions {
```

Comment:
I don't think we need the changes to store the `product` in session storage anymore as the `redirectAfterSignOut` session storage technique negates the need to store `product`?

Reply:
thanks for capturing this, I have removed this